vm/src/frame.rs

```diff
@@ -181,7 +181,7 @@ pub struct Frame {
     pub lasti: RefCell<usize>,        // index of last instruction ran
 }
 
-impl PyObjectPayload2 for Frame {
+impl PyValue for Frame {
```

Comment:
I find myself mixed on this changed - probably just because I'm used to Payload now. I'm sure I'll get used to PyValue.

Reply:
My thinking was that this would be easier to understand for project newcomers - e.g. it feels like it'd be easier to explain that a `PyRef` is a reference to some type that is a `PyValue`.

Reply:
I'm always attached to old names :), but I also feel PyPayload better reflects the fact that it is not a Python value, but a rust inner data for a Python value. That being said, we can always change it later.